src/components/questionary/questionaryComponents/VisitBasis/QuestionaryComponentVisitBasis.tsx

```diff
@@ -1,14 +1,39 @@
+import DateFnsUtils from '@date-io/date-fns';
```

Comment:
Also about `DateFnsUtils` I think it is better to use `LuxonUtils` as it is done here: https://github.com/UserOfficeProject/user-office-frontend/blob/develop/src/components/call/CallGeneralInfo.tsx#L160 and slowly but surely move away from any other library used for dates other than luxon. It is better I think to have only one library for this and use it everywhere.